src/praisonai-agents/praisonaiagents/permissions/command_parser.py

```
@@ -0,0 +1,260 @@
+        args: Positional/flag arguments following the executable.
+        write_targets: Paths that are written/truncated via redirection.
+    """
+
+    executable: str = ""
+    args: List[str] = field(default_factory=list)
+    write_targets: List[str] = field(default_factory=list)
+
+    @property
+    def command_string(self) -> str:
+        """Reconstruct an ``<exe> <args>`` string for glob matching."""
+        parts = [self.executable] + self.args
```

Ruff (0.15.18)

[warning] Consider `[self.executable, *self.args]` instead of concatenation

Replace with `[self.executable, *self.args]`

(RUF005)